src/main/java/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -546,6 +546,13 @@ protected BaseFormParserForJavaRosa(String existingXml, String existingTitle, bo
       // the
       // storage for this form.
       XFormParser exfp = parseFormDefinition(ENCRYPTED_FORM_DEFINITION);
+      // Reset bind element and string length maps since we're actually using another form definition
```

Comment:
"we're actually using another form definition" -> "we're forcing use of a standard form definition with three fields for all encrypted forms" or something like that? Without the context we discussed on Slack I think it would have taken me a bit to look at that constant and figure out what was going on.

Reply:
OK, I'll expand on that

Do you think it would be helpful to add a link to #294?

Reply:
That sounds like a good idea! Or quickly put together a page to address #294 and link to that.